radio/src/gui/128x64/navigation.cpp

```diff
@@ -344,7 +346,7 @@ int checkIncDec(event_t event, int val, int i_min, int i_max, unsigned int i_fla
 #if defined(PCBXLITE)
   if (s_editMode > 0) {
 #endif
-    if (event==EVT_KEY_FIRST(KEY_RIGHT) || event==EVT_KEY_REPT(KEY_RIGHT) || (s_editMode>0 && (IS_ROTARY_RIGHT(event) || event==EVT_KEY_FIRST(KEY_UP) || event==EVT_KEY_REPT(KEY_UP)))) {
+    if (event==EVT_KEY_FIRST(KEY_RIGHT) || event==EVT_KEY_REPT(KEY_RIGHT)) {
```

Comment:
you break SKY9X here

Reply:
Well it disables editing with up/down for it too, but we want that for consistency no?